apps/web/src/jobs/index.ts

```diff
@@ -1,11 +1,24 @@
 import { setupJobs } from '@latitude-data/jobs'
+import env from '$/env'
 
-import env from '../env'
+/**
+ * In theory Nextjs should not pre-render pages that export
+ *   export const dynamic = 'force-dynamic'
+ *
+ * But in practice it does, so we need to check if we are building
+ * the image and avoid connecting to Redis
+ */
+function setup() {
+  if (process.env.BUILDING_CONTAINER) return null
```

Comment:
why do we need this

Reply:
The app tries to connect to Redis while building the bundle